arrowhead-clients/provider.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+import sys
+from sshtunnel import SSHTunnelForwarder
+import requests
+sys.path.insert(1, '/etc/ed7039e/client-library-python')
+import arrowhead_client.api as ar
+
+remote_host = 'tmplt.dev'                    # ip to raspberry pi
+remote_port = 21013                          # port for ssh
+local_host = '127.0.0.1'                     # For the communication with lcm-arrowhead
+local_port = 5005                            # For the communication with lcm-arrowhead
+
+# Creating ssh tunnel for the communication between robot and server
+server = SSHTunnelForwarder(
+   (remote_host, remote_port),
+   ssh_username='root',
+   ssh_private_key='/home/ruben/.ssh/id_rsa',
```

Comment:
Generalize this path, also. If we need adhoc changes to the file system it would be preferrable if they can all be kept under a `/etc/ed7039e/` directory.